src/queue-list/laboratory-patient-list.component.tsx

```diff
@@ -55,19 +49,37 @@ const LaboratoryPatientList: React.FC<LaboratoryPatientListProps> = () => {
 
   const [activatedOnOrAfterDate, setActivatedOnOrAfterDate] = useState("");
 
+  const [filter, setFilter] = useState<
```

Comment:
Thanks @gitcliff. Wanted to inquire if there will be more statuses that will be added so that we could have a better way of adding them in. But as of now LGTM

Reply:
hello @jabahum i got the statuses from this [issue here ](https://openmrs.atlassian.net/browse/TRUNK-5400) and this conversation [here ](https://talk.openmrs.org/t/adding-order-status-to-orders/17928/3?u=gcliff).
Looks like they are the ones we have ,

Reply:
hello @jabahum i have added more statuses as found [here ](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Order.java#L63-L69)